AFQ/api/bundle_dict.py

```diff
@@ -1021,6 +976,12 @@ def is_bundle_in_template(self, bundle_name):
         return "space" not in self._dict[bundle_name]\
             or self._dict[bundle_name]["space"] == "template"
 
+    def _roi_distance_helper(self, roi_or_sl):
```

Comment:
What is this a method of this object? It doesn't seem to be using any of the object attributes.

Reply:
You are correct, in fact I will just make this a lambda function. It's used in one place and is a one-liner, its just passed into 'apply_to_roi_dict'